bundles/org.openhab.binding.pihole/pom.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>4.2.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>org.openhab.binding.pihole</artifactId>
+
+  <name>openHAB Add-ons :: Bundles :: Pi-hole Binding</name>
+
+  <dependencies>
+    <dependency>
```

Comment:
These dependencies should be listed in NOTICE file.

Reply:
It's test scope

Reply:
No one have put mockito in NOTICE file

![obraz](https://github.com/openhab/openhab-addons/assets/2962846/9e3a5ddb-30e7-49e7-bf60-48e82268f250)